### PR TITLE
[NEW] Report > Contract Forecast Menu

### PR DESCRIPTION
**Pull request by mileo**
1. Contract Forecast on Account > Report Menu;
2. New fields: contract_type, price_subtotal_signed/sale/purchase

image

image

### CODE REVIEW

File: contract_forecast/__manifest__.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "license": "AGPL-3",
     "author": "ACSONE SA/NV," "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/contract",
-    "depends": ["base", "contract", "queue_job"],
+    "depends": ["base", "account", "contract", "queue_job"],
```
Comment (len-foss) [inline contract_forecast/__manifest__.py:12]:
```suggestion
    "depends": ["contract", "queue_job"],
```
Contract directly depends on account, so it's redundant. So if you want to change the line, I would rather remove the useless base.

**Review by len-foss — APPROVED** [review]
LGTM with minor comment :+1:

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Review by rvalyi — APPROVED** [review]

**Pull request merged**